spec/like_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  user = User.create(name: 'pierre', photo: 'photo_any_of_course.com', bio: 'bio exist', posts_counter: 0)
+  post = Post.create(author: user, title: 'good foood', text: 'awesome tacos', comments_counter: 0,
+                     likes_counter: 0)
+  subject { Like.new(author: user, post: post) }
+
+  before { subject.save }
+
+  it 'a like is created' do
+    expect(subject).to be_valid
```

Comment:
- [ ] [Optional] [Highly Recommended] It would be better professionally if you add all model testing files inside one directory and you can name it `models`.

Reply:
okay thanks i will do that 😄